java-extensions/hadoop-ext/src/main/java/com/starrocks/fs/StarRocksFileSystem.java

```diff
@@ -0,0 +1,768 @@
+            java.io.FileNotFoundException, org.apache.hadoop.fs.ParentNotDirectoryException,
+            org.apache.hadoop.fs.UnsupportedFileSystemException, java.io.IOException {
+        fs.createSymlink(arg0, arg1, arg2);
+    }
+
+    @Override
+    public org.apache.hadoop.fs.FileStatus getFileLinkStatus(org.apache.hadoop.fs.Path arg0)
+            throws org.apache.hadoop.security.AccessControlException, java.io.FileNotFoundException,
+            org.apache.hadoop.fs.UnsupportedFileSystemException, java.io.IOException {
+        return fs.getFileLinkStatus(arg0);
+    }
+
+    @Override
+    public boolean supportsSymlinks() {
+        return fs.supportsSymlinks();
+    }
+
+    @Override
+    public org.apache.hadoop.fs.Path getLinkTarget(org.apache.hadoop.fs.Path arg0) throws java.io.IOException {
+        return fs.getLinkTarget(arg0);
+    }
+
+    @Override
+    public org.apache.hadoop.fs.FileChecksum getFileChecksum(org.apache.hadoop.fs.Path arg0) throws java.io.IOException {
+        return fs.getFileChecksum(arg0);
+    }
+
+    @Override
+    public org.apache.hadoop.fs.FileChecksum getFileChecksum(org.apache.hadoop.fs.Path arg0, long arg1)
+            throws java.io.IOException {
+        return fs.getFileChecksum(arg0, arg1);
+    }
+
+    @Override
+    public void setVerifyChecksum(boolean arg0) {
+        fs.setVerifyChecksum(arg0);
+    }
+
+    @Override
+    public void setWriteChecksum(boolean arg0) {
+        fs.setWriteChecksum(arg0);
+    }
+
+    @Override
+    public org.apache.hadoop.fs.FsStatus getStatus() throws java.io.IOException {
+        return fs.getStatus();
+    }
+
+    @Override
+    public org.apache.hadoop.fs.FsStatus getStatus(org.apache.hadoop.fs.Path arg0) throws java.io.IOException {
+        return fs.getStatus(arg0);
+    }
+
+    @Override
+    public org.apache.hadoop.fs.Path createSnapshot(org.apache.hadoop.fs.Path arg0, java.lang.String arg1)
+            throws java.io.IOException {
+        return fs.createSnapshot(arg0, arg1);
+    }
+
+    @Override
+    public void renameSnapshot(org.apache.hadoop.fs.Path arg0, java.lang.String arg1, java.lang.String arg2)
+            throws java.io.IOException {
+        fs.renameSnapshot(arg0, arg1, arg2);
+    }
+
+    @Override
+    public void deleteSnapshot(org.apache.hadoop.fs.Path arg0, java.lang.String arg1) throws java.io.IOException {
+        fs.deleteSnapshot(arg0, arg1);
+    }
+
+    @Override
+    public void modifyAclEntries(org.apache.hadoop.fs.Path arg0, java.util.List arg1) throws java.io.IOException {
+        fs.modifyAclEntries(arg0, arg1);
+    }
+
+    @Override
+    public void removeAclEntries(org.apache.hadoop.fs.Path arg0, java.util.List arg1) throws java.io.IOException {
+        fs.removeAclEntries(arg0, arg1);
+    }
+
+    @Override
+    public void removeDefaultAcl(org.apache.hadoop.fs.Path arg0) throws java.io.IOException {
```

Comment:
Do we really need package prefix?

Reply:
it's automatically generated. Can use just need to paste it, no hurt.